beginner_source/audio_preprocessing_tutorial.py

```diff
@@ -62,8 +68,14 @@
 #    STFT function in PyTorch.
 # -  **MuLawEncoding**: Encode waveform based on mu-law companding.
 # -  **MuLawDecoding**: Decode mu-law encoded waveform.
+# -  **TimeStretch**: Stretch stft in time without modifying pitch for a given rate.
```

Comment:
nit: STFT, this should also be changed in documentation :)

but let's write:
"Stretch a spectrogram in time without modifying pitch for a given rate."